projects/components/src/menu-dropdown/menu-item/menu-item.component.scss

```diff
@@ -2,4 +2,9 @@
 
 .menu-item {
   @include popover-menu-item;
+
+  &.disabled {
```

Comment:
hm, we really should make this a global style

Reply:
will do that.

Reply:
wanted to do that. but there are quite a few out there. some without the cursor change. So, need to audit all of them before doing a global change.